public/utils.js

```diff
@@ -120,6 +120,11 @@ define(function (require) {
       }
       return aggConfig;
     },
+    isXYZurl: function (url) {
+      if (!url) return;
```

Comment:
nit-pick
"is" methods should return boolean
```suggestion
      if (!url) return false;
```